src/Dropdown/Dropdown.jsx

```diff
@@ -243,13 +245,27 @@ const Dropdown = ({
   return (
     <StyledFlex flexDirection='column' hasLabel={label} width={1} {...props}>
       {label ? (
-        <Typography
-          as='label'
-          htmlFor={cId}
-          variant='inputLabel'
-          width='fit-content'>
-          {label}
-        </Typography>
+        <Box mb={1}>
+          <Typography
+            as='label'
+            htmlFor={id}
+            variant='base12Bold'
+            color='baseMetal'
+            width='fit-content'>
+            {label}
+          </Typography>
+          {!required && optionalText && (
+            <Typography
+              ml={1}
+              as='label'
+              htmlFor={id}
+              variant='base12Bold'
+              color='baseGray'
+              width='fit-content'>
+              - {optionalText}
+            </Typography>
+          )}
+        </Box>

```

Comment:
I think we should extract this into a component:

```
<Label inputId={id} showOptionalText={!required} optionalText={string?}/>
```

We can set the default prop value for this new Label component to 'Optional' and we don't need to set the default prop in every field that uses it